doc/abrt-action-save-package-data.conf.txt

```diff
@@ -42,8 +42,7 @@ A description of each item follows:
    a program whose basename is equal to one of these, ABRT assigns the crash to the
    executed script instead.
    +
-   Default is `python, python2, python2.7, python3, python3.3, python3.4, python3.5,
-   python3.6, python3.7, perl, perl5.16.2`.
+   Default is `python, perl, php`.
```

Comment:
I am not sure what to put here. `php, perl, python2, python3`?

Reply:
I imagine we would need to hack on support for multiple configuration values, e.g.

```
Interpreters=(perl ([[:digit:]][.][[:digit:]]+[.][[:digit:]])?
Interpreters=php
Interpreters=(platform-)? python ([[:digit:]]([.][[:digit:]])?)?)$
```

And then you could probably just write it as a list.

Reply:
Meh, that looks ugly and complicated.

Reply:
It’s as ugly and as complicated as the regular expressions are, really, but I see your point.

Reply:
Okay, counter-proposal(s):

1. Hardcode meta-values, like `@perl` and `@python` and reference those in a comma-separated list, like before
2. Allow to configure such meta-values in the config (allowing regular expressions) and do 1